packages/tocco-ui/src/EditableValue/typeEditors/Document.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Document = props => {
+  return (
+    <div className="form-control-static">
+      <a href={props.value.binaryLink} download>
+        {props.value.fileName}
+      </a>
+    </div>
+  )
+}
+
+Document.propTypes = {
+  value: React.PropTypes.object
```

Comment:
-  `shape` should be used instead of `object`
- should be marked as required (otherwise null should be handled so that there are no null pointers at line 6)?